proposed/phpdoc-tags.md

```diff
@@ -693,16 +690,10 @@ function or method. When provided, it MUST contain a "Type"
 to indicate what is returned; the description on the other hand is OPTIONAL yet
 RECOMMENDED in case of complicated return structures, such as associative arrays.
 
-The @return tag MAY have a multi-line description and does not need explicit
-delimiting.
-
-It is RECOMMENDED to use this tag with every function and method. An exception to
-this recommendation, as defined by the Coding Standard of any individual project,
-MAY be:
-
-   **functions and methods without a `return` value**: the @return tag MAY be
-   omitted here, in which case an interpreter MUST interpret this as if
-   `@return void` is provided.
+It is RECOMMENDED to use this tag with every function and method.
```

Comment:
Should we not rephrase this to:

It is RECOMMENDED to use this tag with every function or method where PHP native types are insufficent.